bubblesub/cmd/focus_widget.py

```diff
@@ -51,6 +52,26 @@ class FocusWidgetCommand(BaseCommand):
     names = ["focus-widget"]
     help_text = "Focuses given widget."
 
+    @property
+    def is_enabled(self) -> bool:
+        target = str(self.args.target)
+        if target == "spectrogram" and (
+            self.api.gui.current_view == ViewState.Audio
+            or self.api.gui.current_view == ViewState.Full
+        ):
+            return True
+
+        if (
+            target in ("console", "console-input")
+            and self.api.gui.current_view == ViewState.Full
+        ):
+            return True
+
+        if target in ("text-editor", "note-editor", "subtitles-grid"):
+            return True
+
+        return False
```

Comment:
This is completely broken,  think what happens for `MarginLeftEditor`. It should work like a blacklist rather than whitelist.
Why do you cast target to str?! These tests should compare against `_TargetWidget` members.